src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.ts

```diff
@@ -52,6 +51,7 @@ const verifyV2 = async (
   const { issuers } = data;
 
   const revocation: (v2.Revocation | undefined)[] = issuers.map((issuer) => issuer.revocation);
+  // TODO unreachable
```

Comment:
I added this kind of comments for parts of the code that should not be reachable anymore

Reply:
why TODO?

Reply:
wondering if we keep (in case) or remove